src/processor/processor.module.ts

```diff
@@ -47,10 +50,20 @@ import { ProviderWebhookService } from './provider-webhook.service';
         backoff: {
           type: 'exponential',
         },
-        removeOnComplete: true,
-        removeOnFail: true,
+        removeOnComplete: false,
```

Comment:
This would keep all the messages around and none of them would be cleaned up in redis. I think maybe having a cap on the number of messages stored would be better.

Reply:
true may not need to keep the completed ones, they are on chain anyways